downstream/assemblies/platform/assembly-ug-controller-workflow-job-templates.adoc

```diff
@@ -14,11 +14,12 @@ These resources include the following:
 * Project syncs
 * Inventory source syncs
 
-The *Templates* page shows the workflow and job templates that are currently available. 
-The default view is collapsed (Compact), showing the template name, template type, and the statuses of the jobs that have run by using that template. 
-You can click the arrow next to each entry to expand and view more information. 
+The *Automation Templates* page shows the workflow and job templates that are currently available. 
+The default view is to show each template as a card, showing the template name and template type. 
+Select the template name to display more information about the tenmplate, including when it  last ran.
```

Comment:
Typo "tenmplate" -> "template"

in the phrase "to display more information about the tenmplate,"